src/platforms/dotnet/guides/aspnet/performance/troubleshooting.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: Troubleshooting
+sidebar_order: 100
+description: "Learn more about how to troubleshoot performance issues with the ASP.NET SDK."
+---
+
+## Transactions get grouped together as `GET /*/*.`
+
+The SDK creates the transaction name as follows
+
+```csharp
+var method = httpContext.Request.HttpMethod;
+var path = httpContext.Request.Path;
+
+var transactionName = $"{method} {path}";
+
+// Since the name is derived from the path and potentially contains identifiers it is marked as such
+transactionContext.NameSource = TransactionNameSource.Url;
+```
+
+Since the `URL` might contain potential identifiers these transactions have their name stripped.
+It is possible to work around this and still rely on automatic performance instrumentation. You can access the currently active transaction and overwrite the `Name` and `NameSource`.
```

Comment:
```suggestion
It is possible to work around this and still rely on automatic performance instrumentation. You can access the currently active transaction and overwrite the `Name` and `NameSource`. Specifying the exact name of the transaction, without any variables. Using the example above, it would look like:
```